buildscripts/common.mk

```diff
@@ -16,7 +16,7 @@
 .PHONY: check_license-go
 check-license-go:
 	@echo ">> checking license header"
-	@licRes=$$(for file in $$(find . -type f -iname '*.go' ! -path './vendor/*' ) ; do \
+	@licRes=$$(for file in $$(find . -type f -iname '*.go' ! -path './vendor/*' ! -path './pkg/debug/debug.go' ! -path './pkg/debug/release.go' ) ; do \
```

Comment:
It will be better if we exclude `./pkg/debug/` completely instead of single files, since list may increase in future.